### PR TITLE
Release notes for Private Cloud Operator 1.1.1

### PR DESCRIPTION
**Pull request by zlogic**
New release of the Mendix Operator:
* new supported databases
* some common questions and remarks for existing databases and file storage
* how to upgrade to the latest Operator

We're hoping to release this on June 19 afternoon.

### CODE REVIEW

File: content/developerportal/deploy/private-cloud-cluster.md [hunk 1]
```diff
@@ -181,25 +181,69 @@ When you reconfigure your namespace with databases or storage, you will add new
...
+{{% alert type="info" %}}
+Configuration parameters will not validated and will be provided to the Mendix app as-is. If the arguments are not valid or there's an issue with permissions, the Mendix Runtime will fail to start the and deployment will appear to hang with **Replicas running** and **Runtime** showing a spinner.
+{{% /alert %}}
+
+{{% alert type="info" %}}
+If the plan name already exists you will receive an error that it cannot be created. This is not a problem, you can continue to use the plan, which will now have the new configuration.
+{{% /alert %}}
+
+{{% alert type="info" %}}
+To use this plan, [upgrade](/developerportal/deploy/private-cloud-upgrade-guide) the Mendix Operator to version 1.1.0 or later.
```
Comment (MarkvanMents) [inline content/developerportal/deploy/private-cloud-cluster.md:237]: @zlogic is this really 1.1.0, or should it be 1.1.1?

**Comment by MarkvanMents** [comment]
@zlogic - I've reviewed this and can publish it whenever you do the release.

File: content/developerportal/deploy/private-cloud-cluster.md (hunk 1 above)
Comment (zlogic) [reply content/developerportal/deploy/private-cloud-cluster.md:237]: @MarkvanMents the storage plans are technically compatible with any version of the Operator - but only Operator 1.1.0 (and later) can check if the plan is used by another environment and prevent two apps from fighting over a database.

**Comment by zlogic** [comment]
@MarkvanMents Can you please publish these changes now?

**Pull request merged**